### PR TITLE
[MBO-664][MBO-665][MBO-660] Improve module catalog

### PR DESCRIPTION
**Pull request by julienr114**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 2.0.x
| Description?      | Improve catalog page. Cards. Add and change wording, fix some element position and margin.
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #MBO-664 #MBO-665 #MBO-660
| How to test?      | Install the module on a shop
| Possible impacts? |

### CODE REVIEW

File: ps_mbo.php [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?php
+
```
Comment (sowbiba) [inline ps_mbo.php:2]: to be removed ;)

File: src/Controller/Admin/ModuleController.php [hunk 2]
```diff
@@ -0,0 +1,241 @@
+<?php
+
```
Comment (sowbiba) [inline src/Controller/Admin/ModuleController.php:2]: No line break

File: src/Controller/Admin/ModuleController.php [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
```
Comment (sowbiba) [inline src/Controller/Admin/ModuleController.php:21]: No line break needed

**Review by sowbiba — COMMENTED** [review]
👏🏾

File: views/templates/admin/controllers/module_catalog/Includes/module_actions_buttons.html.twig [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        'name': name,
+        'classes_form': 'btn-block',
+        'classes' : 'btn btn-secondary btn-block',
+        'url': urls[url_active],
+        'action': url_active,
+        'transDomain': transDomains}
+    %}
+    {% if (urls|length > 1) %}
+        <input type="hidden" class="btn" /> 
+        AAA
```
[the remark this answers is not in the document]
Comment (julienr114) [reply views/templates/admin/controllers/module_catalog/Includes/module_actions_buttons.html.twig]: bien vu

**Review by sowbiba — APPROVED** [review]

**Pull request merged**